grammar.js

```diff
@@ -1628,8 +1628,9 @@ module.exports = grammar({
         $.keyword_partition,
       ),
       choice(
-        seq('(', $.identifier, ')'), // postgres
-        $.column_definitions, // impala/hive
+        // seq('(', $.identifier, ')'), // postgres
+        seq('(', $.identifier, repeat(seq(',', $.identifier)), ')'), // postgres & Impala (CTAS)
```

Comment:
you could use `comma_list` for this possibly but I'm still not sure what was going on with those in conflicts in #150 . Also the comment on line 1631 can be cleared

Reply:
I can also use `paren_list`. I forgot that they exist :-) Thanks for the hint.